compare-images

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+
+import json
+import optparse
+import os
+import sys
+from tabulate import tabulate
```

Comment:
I generally prefer `import tabulate` and `tabulate.tabulate(...)`, to make it clear when a function comes from a pkg and not from the current file.   WDYT?